modules/parent-join/src/main/java/org/elasticsearch/join/mapper/ParentJoinFieldMapper.java

```diff
@@ -114,10 +114,10 @@ private static void checkParentFields(String name, List<ParentIdFieldMapper> map
         }
     }
 
-    static class Builder extends FieldMapper.Builder<Builder, ParentJoinFieldMapper> {
+    public static class Builder extends FieldMapper.Builder<Builder, ParentJoinFieldMapper> {
```

Comment:
ah right, I think it's ok since it's the main mapper. We should just hide the other mapper builders since they are only used internally